package.json

```diff
@@ -401,6 +409,16 @@
         "title": "Hide Generated",
         "category": "ObjectScript"
       },
+      {
+        "command": "vscode-objectscript.explorer.showSystem",
+        "title": "Show System",
```

Comment:
Since %Z classes aren't system classes, I suggest this menu item name be "Show % Classes"

reference: https://docs.intersystems.com/irislatest/csp/docbook/Doc.View.cls?KEY=GORIENT_appx_identifiers

Reply:
%ZEN also %Z